roles/openshift_hosted/README.md

```diff
@@ -55,6 +63,11 @@ Example Playbook
       cafile: /path/to/my-router-ca.crt
     openshift_hosted_router_registryurl: 'registry.access.redhat.com/openshift3/ose-haproxy-router:v3.0.2.0'
     openshift_hosted_router_selector: 'type=infra'
+    openshift_hosted_registry_storage_kind=glusterfs
+    openshift_hosted_registry_storage_glusterfs_path=external_registry_path
+    openshift_hosted_registry_storage_glusterfs_endpoints=external-gluster-ep-name
```

Comment:
You don't need to set this. The default of `glusterfs-registry-endpoints` should suffice, since our constraints make it such that we don't allow a conflict.